wolfcrypt/src/aes.c

```diff
@@ -14906,4 +14914,278 @@ int wc_AesEaxFree(AesEax* eax)
 
 #endif /* WOLFSSL_AES_EAX */
 
+#ifdef WOLFSSL_AES_CTS
+
+
+/* One-shot API */
+int wc_AesCtsEncrypt(const byte* key, word32 keySz, byte* out,
+                     const byte* in, word32 inSz,
+                     const byte* iv)
+{
+#ifdef WOLFSSL_SMALL_STACK
+    Aes *aes = NULL;
+#else
+    Aes _aes;
```

Comment:
Prefer `Aes aes[1]`. Single line that does the same.

Reply:
Was following other functions in `aes.c` but a single element array is cleaner.